operator/src/main/resources/scripts/modelInImage.sh

```diff
@@ -679,7 +679,11 @@ function wdtCreatePrimordialDomain() {
     > ${WDT_OUTPUT}
   ret=$?
   if [ $ret -ne 0 ]; then
-    trace SEVERE "WDT Create Domain Failed ${ret}"
+    if [ "JRF" == "$WDT_DOMAIN_TYPE" ] ; then
+      trace SEVERE "MII Fatal Error: WDT Create Domain Failed ${ret}"
```

Comment:
Add comment that "MII Fatal Error" is a keyword that the operator detects, and if detected, the operator will suppress introspection retries.

Reply:
added

Reply:
How about:
    # MII Fatal Error is handled by DomainProcessorImpl.isShouldContinue
-->
    # The string "MII Fatal Error" is detected by DomainProcessorImpl.isShouldContinue